src/components/profile/ProfileSetProfile.jsx

```diff
@@ -20,7 +20,8 @@ export default function ProfileSetProfile() {
       manual: true,
       onSuccess: async () => {
         toast.success("Profile updated");
-        navigate("../account", { replace: true });
+        navigate("../", { replace: true });
+        navigate(0);
```

Comment:
what do this do?
the website refreshes twice. remove this and make sure the website only loads once after the change is submitted.